README.md

```diff
@@ -128,6 +128,7 @@ Asynchronous method that runs a site wide scan. Options come in the form of an o
 - `port` (number) - When the `path` is provided as a local path on disk, the `port` on which to start the temporary web server.  Defaults to a random high range order port.
 - `recurse` (boolean) - By default, all scans are shallow.  Only the top level links on the requested page will be scanned.  By setting `recurse` to `true`, the crawler will follow all links on the page, and continue scanning links **on the same domain** for as long as it can go. Results are cached, so no worries about loops.
 - `linksToSkip` (array) - An array of regular expression strings that should be skipped during the scan.
+- `linksAttributes` (object) - Expand tag attributes to find URLs. Keys are attributes and values are array of tag to search.
```

Comment:
As mentioned, I think we could just drop this to be honest, and always parse `style`

Reply:
This is not for style its for custom `data-` attributes. There's no way to standatize it at all.